lib/adapters/mongodb/sse/index.js

```diff
@@ -153,15 +153,31 @@ module.exports = function (mongodbOplogUrl, options) {
         return mongoEntity
     }
 
+    function uncollapseSetObject(obj) {
```

Comment:
The dot-object lib can do the reverse 'dot' to 'object' as well : https://www.npmjs.com/package/dot-object#expand-to-an-object